src/components/recipe-list.tsx

```diff
@@ -0,0 +1,20 @@
+type Props = {
+  recipeText: string;
+  annotation: string;
+  orderNum: number;
+};
+
+// レシピリスト
+export const RecipeListItem = ({ recipeText, annotation, orderNum }: Props) => {
+  return (
+    <div className="flex px-[12px] py-[8px] border-b-[1px]">
+      <div className="mr-[8px]">
+        <span className="flex justify-center w-[18px] h-[18px] rounded-full bg-[#CA3214] text-white text-[12px]">{orderNum}</span>
+      </div>
+      <div>
+        <p className="text-[14px] leading-4 mb-[8px]">{recipeText}</p>
+        <p className="text-[10px] text-[#86848D]">※{annotation}</p>
+      </div>
+    </div>
```

Comment:
すみません、大きくスタイル変更しましたが、必要に応じて適宜変更したほうが良いと思いました！

```suggestion
 <div className="flex items-start gap-x-2 py-2 border-y">
      <div className="mt-px flex items-center justify-center h-5 w-5 text-sm shrink-0 select-none rounded-full bg-accent text-accent-foreground">
        {stepNumber}
      </div>
      <div className="flex-1 leading-snug">{recipeText}</div>
    </div>
```

以下変更点です！

- 具体的な数値を指定する代わりに、ユーティリティクラスを使用
    - デザインの一貫性を保つことが目的です！
    - もちろんデザインによっては仕方がないケース（具体的な数値を指定するケース）もあると思っています！
-  カラーテーマの指定
    - [shadcn-plugin.ts](https://github.com/qin-team-recipe/12-recipe-app/blob/main/src/lib/shadcn-plugin.ts)にカラー定義まとめているので、極力ここから指定していただければと思います！